### ISSUE TITLE
Onboarding

### ISSUE THREAD

**Issue by will-kyaria** [willmbennett/kyaria#135]
Todo:
- [ ] Change profile to be simpler, leave all the complex data structures on the resume
- [ ] Switch to piecemeal onboarding. Aka link to various pages instead of having all the onboarding on one page
- [ ] Start small and add back one section at a time, only when 100% confident in the page. 

Onboarding sections to add back in:
- [ ] Resume (highest priority, link to resume builder)
- [ ] LinkedIn bio (high priority)
- [ ] Cover Letter (high priority, but blocked by cover letter builder)
- [ ] Interview prep (lower priority)
